docs/modules/ROOT/pages/Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/Phone Home/PhoneHome.adoc

```diff
@@ -65,9 +45,59 @@ would only gather data specific to Payara and has no contact with any
 running applications, or indeed anything on your machine other than the
 mentioned attributes.
 
+== Disabling and Enabling PhoneHome
+
+The phone home service is enabled by default when starting a Payara
+Server or Payara Micro. The phone home
+service can easily disabled in four different ways.
+
+[[enable-phone-home]]
+== Enable the Phone Home service
+
+====
+asadmin> enable-phone-home
+====
+
+[[disable-phone-home]]
+== Disable the Phone Home service
+
+[[disable-with-asadmin]]
+=== Using Asadmin
+
+Run this asadmin command then restart the server to disable PhoneHome
+
+====
+asadmin> disable-phone-home
+====
+
+[[disable-with-domainxml]]
+=== Disabling in the _domain.xml_ File
+
+- Open `domain.xml` with a text editor
+- Find the tag `<config name="server-config">
+- Within that tag, add the following tag:
+- <phone-home-runtime-configuration enabled="false"/>
+
+[[removing-the-service-module]]
+=== Disable by Removing The Service Module
+
+- Navigate to the `${PAYARA_INSTALL_DIR}/glassfish/modules` directory
+- Delete the `phonehome-bootstrap.jar` module.
+
+[[list-phone-home]]
+== List the current status of the Phone Home service.
+
+====
+asadmin> list-phone-home
+====
+
 [[source-code]]
-== Source Code
+=== Source Code
```

Comment:
The link to the source code can be moved to the See Also section and this section can be deleted.